internal/testutils/args.go

```diff
@@ -27,7 +27,7 @@ func IsVerbose() bool {
 			if !ok {
 				continue
 			}
-			isVerbose = value == "true"
+			isVerbose = value != "false"
```

Comment:
How about we use [`testing.Verbose()`](https://pkg.go.dev/testing#Verbose) instead of our custom `testutils.IsVerbose`?